### ISSUE TITLE
Accessibility: title or text is missing from document link to announce new tab

### ISSUE THREAD

**Issue by VladimirAus** [swagger-api/swagger-ui#7818]
When a user navigates to the link under API Specification, it is not read out that the link opens in a new window. This would cause confusion to screen reader users who would not expect focus to move into a new window. 

* **Browser/AT used:**    Chromium Edge 92 + JAWS 2020 / Firefox 86 + NVDA 2020.3
* **Success Criteria:**  [3.2.2 On Input (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=321%2C332#predictable)

**Comment by mgifford** [comment]
Can you provide the html @VladimirAus that would be easier to see what it is vs what you would like it to be?   
  
In general titles on links isn't a great idea as it is so poorly supported.
